src/darwin/frida-helper-backend-glue.m

```diff
@@ -1,4 +1,5 @@
 #include "frida-helper-backend.h"
+#include "frida-tvos.h"
```

Comment:
```suggestion

#include "frida-tvos.h"
```

(Includes are grouped, first own header (if present), then any internal headers, and finally any system/external headers.)